tests/TestCase/Validation/ValidatorTest.php

```diff
@@ -563,7 +563,7 @@ public function testErrorsWithLastRule()
         $errors = $validator->errors(['email' => 'not an email!']);
         $expected = [
             'email' => [
-                'alpha' => 'The provided value is invalid'
+                'alpha' => 'alpha'
```

Comment:
Couldn't the basic app skeleton feature a cake-validation "domained" locale file that helps with the default validations and rule messages?